Doc/library/shutil.rst

```diff
@@ -470,42 +470,7 @@ file then shutil will silently fallback on using less efficient
 copytree example
 ~~~~~~~~~~~~~~~~
```

Comment:
I think removing it would make sense but you leave a hanging heading. Do you have a reason for this?

Also, the bpo asked for a fix to the example, I would think deciding to remove it instead requires some consensus, maybe @willingc and @Mariatta can advise.

Reply:
I don't think this leaves the header dangling - it relates to the next example (not that the last modified like changes "Another example..." to "An example...", because the next one is now the first).